packages/mobile/src/redux/migrations.ts

```diff
@@ -41,4 +41,14 @@ export const migrations = {
       },
     }
   },
+  3: (state: any) => {
```

Comment:
This just adds new state right? No need for a new migration. Also if you did add one, you'd have to also rev the version in redux/store.ts and preferably also create a test